README.md

```diff
@@ -79,6 +80,14 @@ Use `toUpperCase()` to capitalize first letter, `slice(1)` to get the rest of th
 var capitalize = str => str[0].toUpperCase() + str.slice(1);
 ```
 
```

Comment:
Thank you very much for your contribution! This is a great fit for the list.

Please move all your changes to a file inside the snippets folder, named `capitalize-first-letter-of-every-word.md`. We keep a list of all snippets in separate files for easier maintenance. After I rebuild using the building script, it'll all be exactly the same as it is now.